hybrasyl/Objects/User.cs

```diff
@@ -2536,7 +2563,23 @@ public override bool UseCastable(Xml.Castable castObject, Creature target = null
                 SendSystemMessage(restrictionMessage);
                 return false;
             }
-            
+
+            if (castObject.Script != null)

```

Comment:
similarly here, scripting is already handled by `Creature.UseCastable`, and `User` calls `base.UseCastable` - this is also duplicate code?

Reply:
so on this one, without this added the script would never fire.